LatyshevEV/lab1/Code

```diff
@@ -0,0 +1,48 @@
+#include<iostream>
+#include<string>
+using namespace std;
+int main()
+{
+	setlocale(0, "");
+	float left;
+	char sign;
+	float right;
+	bool flag = 0;
+	cout << "Левый операнд пожалуйсто" << endl;
```

Comment:
Use more formalistic text style
We are not at "KVN" show
The same for output strings below